test/integration/test_composite_job.py

```diff
@@ -1,3 +1,7 @@
+# pylint: disable-all
+# type: ignore
+# TODO: Reenable and fix integration tests.
```

Comment:
I want to work on enabling and fixing the integration tests in a separate PR because this PR has gotten too big already :).